src/MatrixTools.js

```diff
@@ -24,5 +25,34 @@ module.exports = {
     getDisplayAliasForRoom: function(room) {
         return room.getCanonicalAlias() || room.getAliases()[0];
     },
+
+    isDirectMessageRoom: function(room, me, ConferenceHandler, hideConferenceChans) {
```

Comment:
Heads up that this function will change shortly once we've finally finally defined how to track which rooms are DMs in terms of room or user configuration state.  This is fine for now though.